packages/fxa-shared/gaurds/AdminPanelGuard.ts

```diff
@@ -0,0 +1,96 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Typo in folder name, `gaurds` -> `guards`

Reply:
_Turns I don't know how to spell guard... 😥_